resources/lang/en/trans.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Authentication Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used during authentication for various
+    | messages that we need to display to the user. You are free to modify
+    | these language lines according to your application's requirements.
+    |
+    */
+
+    'Email address' => 'Email address',
+    'Password' => 'Password',
+    'Log in' => 'Log in',
+    'Register' => 'Register',
+    'Search' => 'Search',
+    'Profile' => 'Profile',
+    'Log out' => 'Log out',
+    'Email or Pasword not matched!' => 'Email or Pasword not matched!',
```

Comment:
```suggestion
    'email_password_not_matched' => 'Email or Pasword not matched!',
```
Nên đặt dạng _ giữa các từ và cố gắng bớt từ sao cho ngắn nhất có thể